docs/semgrep-code/editor.md

```diff
@@ -188,6 +188,31 @@ Once you've written a rule and created comment annotations, you can run your rul
 
 <EnableTurboMode />
 
+## Code search (beta)
+
+Code search allows you to test a Semgrep rule by running it against one or more GitHub repositories instead of just a few lines of test code. Its results highlight all instances of matching code in your repositories, allowing you to see whether your rule works as intended or not. This rapid feedback can help you develop more accurate and effective rules.
```

Comment:
```suggestion
Code search allows you to test a Semgrep rule by running it against one or more GitHub repositories or projects instead of just a few lines of test code. Its results highlight all instances of matching code in those target repositories, allowing you to see whether your rule works as intended or not. This rapid feedback can help you develop more accurate and effective rules.
```